This changes help to support Virgl(Virtual 3D GPU) for

renchenglei opened the pull request:
guest virtual machines.

Tracked-On: OAM-OAM-87916
Signed-off-by: Jin, Zhi <zhi.jin@intel.com>

sysopenci reviewed (commented):
Autobuild started from pull-request-changes on this PR.

FAILURE: CheckBug Bad comments/Bugs

For more information, see: /absp/builders/celadon-autobuild/builds/1429

renchenglei committed:
Enable Virgl(Virtual 3D GPU)

This changes help to support Virgl(Virtual 3D GPU) for
guest virtual machines.

Tracked-On: OAM-88714
Signed-off-by: Jin, Zhi <zhi.jin@intel.com>

renchenglei commented:
@YuanjunHuang, please help take a look, thanks a lot!

renchenglei commented:
@feijiang1, please help take a look, thanks a lot!

sysopenci reviewed (commented):
Failure when executing preintegration builder

FAILURE:CheckApprovals Some checks are failed. See the step results.html

More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/1273

sysopenci reviewed (commented):
Failure when executing preintegration builder

FAILURE:CheckApprovals Some checks are failed. See the step results.html

More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/1274

sysopenci reviewed (commented):
Failure when executing preintegration builder

FAILURE:CheckBug Bad comments/Bugs

More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/1275

sysopenci reviewed (commented):
Failure when executing preintegration builder

FAILURE:CheckApprovals Some checks are failed. See the step results.html

More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/1276

sysopenci reviewed (commented):
PR integrated into 'preintegration' builder



More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/1277

Done by "Ren, Chenglei" <chenglei.ren@intel.com>

sysopenci reviewed (commented):
PR integrated into 'premerge' builder



More details, logs, and binaries: /absp/builders/celadon-premerge/builds/647

Done by "Ren, Chenglei" <chenglei.ren@intel.com>

sysopenci reviewed (commented):
cactus finished building and testing your changes 

It was requested for merge by "Ren, Chenglei" <chenglei.ren@intel.com> 

so we merge it. Thanks for your contribution.

more details, logs, and binaries: /absp/builders/celadon-mergerequest/builds/1344
